instructions/index.js

```diff
@@ -0,0 +1,15 @@
+let today = new Date();
+let thisYear = today.getFullYear();
+let footer = document.querySelector("footer");
+let copyright = document.createElement("p");
+copyright.innerText = "Vanessa Rodriguez " + thisYear;
+footer.appendChild(copyright);
+
+const skills = ["Patient Assessment", "HTML", "CSS", "JavaScript", "Medical Terminology"];
+let skillsSection = document.querySelector("skills");
+let skillsList = document.getElementById("skills")
+for (skill_listed in skills) {
```

Comment:
Make sure you use camel case (`skillListed`) instead of snake case (`skill_listed`) in JS